src/test/java/usecases/pull_game_ended/PgeInteractorTests.java

```diff
@@ -0,0 +1,49 @@
+package usecases.pull_game_ended;
+
+import entities.Player;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+public class PgeInteractorTests {
+
+    // PgeInputBoundary, PgeInputData used to test PgeInteractor
+    private PgeInputBoundary pgeib;
+    private PgeInputData pgeid;
+
+    @Before
+    public void setUp() {
+        // Add players for a list of players
+        Player p1 = new Player("p1", "1");
+        Player p2 = new Player("p2", "2");
+
+        // Instantiate pgeInputBoundary
+        pgeib = new PgeInteractor(new PgeOutputBoundary() {
+            @Override
+            public void notifyGameEnded(PgeOutputData data) {}
```

Comment:
We should collect the ids from the responses and use this data in tests to check that the players' ids are the same.